src/metatrain/__main__.py

```diff
@@ -81,7 +81,7 @@ def main():
     if callable == "train_model":
         # define and create `checkpoint_dir` based on current directory, date and time
         checkpoint_dir = _datetime_output_path(now=datetime.now())
-        os.makedirs(checkpoint_dir)
+        os.makedirs(checkpoint_dir, exist_ok=True)  # exist_ok=True for distributed
```

Comment:
Hmmm perhaps here it's better to have the directory created by the main process (and then synchronize?)

Reply:
I think that `exists_ok` is fine here. There is no risk of data loss if two process create the directory at the same time